tools/internal_ci/linux/grpc_interop_google_default_toprod.cfg

```diff
@@ -0,0 +1,30 @@
+# Copyright 2018 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Config file for the internal CI (in protobuf text format)
+
+# Location of the continuous shell script in repository.
+build_file: "grpc/tools/internal_ci/linux/grpc_run_interop_tests.sh"
+timeout_mins: 60
+action {
+  define_artifacts {
+    regex: "**/sponge_log.xml"
+    regex: "github/grpc/reports/**"
+  }
+}
+
+env_vars {
+  key: "RUN_TESTS_FLAGS"
+  value: "-l c++ --transport_security google_default_credentials --cloud_to_prod --cloud_to_prod_auth --prod_servers default gateway_v4 --use_docker --internal_ci -t -j 8"
```

Comment:
Is it possible to include this test in existing interop test instead of creating a new set of configs? By the way, you'll need corresponding Kokoro configs in order for this to run. That's why adding to the existing job is more desirable.

Reply:
I am fine with including the test into current cloud-to-prod interop test. But the current interop test (grpc_run_interop_tests.sh) is configured to execute run_interop_tests script only once with the existing flag `RUN_TESTS_FLAGS`, and I do not know how to execute it twice without modifying grpc_run_interop_tests.sh by introducing another one-off flag.

Reply:
Or maybe I can modify run_interop_tests script to add another job for `cloud_to_prod` and `cloud_to_prod_auth` with google default credentials. How do you think?